cilium-ubuntu-4.19.json

```diff
@@ -66,6 +66,9 @@
       ]
     },{
       "type": "shell",
+      "environment_vars": [
+          "ARCH=amd64",
```

Comment:
I think the build failures are due to the trailing comma here (same for other builds in their respective `.json` files).

Reply:
Thanks, fixed!

Reply:
Also had to change to use "VM_ARCH" to not confuse some compilations/intalls.